cmd/csi-addons/main.go

```diff
@@ -122,6 +123,8 @@ func getKubernetesClient() *kubernetes.Clientset {
 		panic(err.Error())
 	}
 
+	config.ContentType = runtime.ContentTypeProtobuf
```

Comment:
can you please check again, i see we are creating client in 3 places https://github.com/search?q=repo%3Acsi-addons%2Fkubernetes-csi-addons%20kubernetes.NewForConfig&type=code

Reply:
Yes, that's what I said above that in other 2 places the client is used to interact with CRDs. That's the reason we can't use the protobuf encoding.